clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -148,11 +148,11 @@ private AbstractRequestResponse createResponseHeader() {
     }
 
     private AbstractRequest createConsumerMetadataRequest() {
-        return new ConsumerMetadataRequest("test-group");
+        return new GroupMetadataRequest("test-group");
     }
 
     private AbstractRequestResponse createConsumerMetadataResponse() {
-        return new ConsumerMetadataResponse((short) 1, new Node(10, "host1", 2014));
+        return new GroupMetadataResponse((short) 1, new Node(10, "host1", 2014));
```

Comment:
Seems like it's just testing serialization, so 1 was probably chosen for convenience. I'll change to Errors.NONE instead.